postgres-master/postgresql.conf.ctmpl

```diff
@@ -17,6 +17,7 @@ log_autovacuum_min_duration = '250'
 log_line_prefix = '%m %u@%d %p %r '
 log_lock_waits = 'on'
 log_min_duration_statement = '1000'
+log_temp_files = 'off'
```

Comment:
https://www.postgresql.org/docs/9.5/runtime-config-logging.html says that this is an integer

>Controls logging of temporary file names and sizes. Temporary files can be created for sorts, hashes, and temporary query results. A log entry is made for each temporary file when it is deleted. A value of zero logs all temporary file information, while positive values log only files whose size is greater than or equal to the specified number of kilobytes. The default setting is -1, which disables such logging. Only superusers can change this setting.